schemas/interfaceType.schema.tpl.json

```diff
@@ -0,0 +1,4 @@
+{
+  "_type": "controlledTerms:InterfaceType",
```

Comment:
We should discuss the schema name. Maybe this is too ambiguous? Not sure maybe InteractionInterfaceTypes ? This could then also include VUI besides, GUI, API and CLI